docs/topics/wildfly-swarm/docs/reference/fractions/undertow.adoc

```diff
@@ -737,11 +806,14 @@ swarm.undertow.servers._KEY_.https-listeners._KEY_.max-processing-time::
 The maximum processing time taken by a request on this listener
 
 swarm.undertow.servers._KEY_.https-listeners._KEY_.no-request-timeout:: 
-The length of time in milliseconds that the connection can be idle before it is closed by the container, defaults to 60000 (one minute)
+The length of time in milliseconds that the connection can be idle before it is closed by the container.
 
 swarm.undertow.servers._KEY_.https-listeners._KEY_.processing-time:: 
 The total processing time of all requests handed by this listener
 
+swarm.undertow.servers._KEY_.https-listeners._KEY_.proxy-address-forwarding:: 
+Enables  handling of x-forwarded-host header (and other x-forwarded-* headers) and use this header information to set the remote address. This should only be used behind a trusted proxy that sets these headers otherwise a remote user can spoof their IP address.
```

Comment:
header values should use backticks


Also, there is a double space here:
> Enables  handling of

Reply:
Note to self: not easily fixable in our sources.